src/python/pants/backend/codegen/thrift/apache/python/subsystem.py

```diff
@@ -25,11 +27,28 @@ def register_options(cls, register):
                 "See `thrift -help` for supported values."
             ),
         )
+        register(
+            "--runtime-dependencies",
```

Comment:
Not for this change, but it would be nice if this could automatically default to the right thing based on the expected version in the ThriftSubsystem.

Reply:
It would be nice. None of the codegen tools have the ability to do this currently though. Part of the complication would be how it should interact with multiple resolves.